.gitignore

```diff
@@ -3,3 +3,4 @@ composer.phar
 composer.lock
 /src/views
 /src/config/config.dev.php
+.idea
```

Comment:
Thank for the suggestion, I go to revert this change to a preview commit of file.